src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,39 @@
+package racingcar.domain;
+
+import static java.util.stream.Collectors.toList;
+
+import java.util.List;
+import racingcar.domain.random.RandomNumberGenerator;
+import racingcar.domain.result.MidtermResult;
+import racingcar.domain.result.WinnersResult;
+
+public class Cars {
+
+    private final List<Car> cars;
+    private final RandomNumberGenerator randomNumberGenerator;
+
+    public Cars(List<String> names, RandomNumberGenerator randomNumberGenerator) {
+        cars = names.stream().map(Car::of).collect(toList());
```

Comment:
stream 함수와 같이 체이닝 메서드를 사용하는 경우 가독성을 위해 아래와 같이 개행하여 사용합니다 :)

```suggestion
        cars = names.stream()
            .map(Car::of)
            .collect(toList());
```

Reply:
이하 다른 코드에 위와 동일한 부분에 대해서는 추가적인 코멘트는 남겨두지 않도록 할게요!! 전체적으로 코드를 한번 확인해보면 좋을 것 같아요!!

Reply:
변경했습니다🔧